src/api/objects/ObjectAPI.js

```diff
@@ -232,7 +238,7 @@ export default class ObjectAPI {
       .get(identifier, abortSignal)
       .then((domainObject) => {
         delete this.cache[keystring];
-        if (!domainObject && abortSignal.aborted) {
+        if (!domainObject && abortSignal?.aborted) {
```

Comment:
do we need this? we should probably prefer to fail hard in the API, if anywhere, right?

Reply:
This code was already in and I'd rather not touch it any more right now since the `get` is sort of core to the app.

Reply:
It says the change is added by this PR though?